docs/en/latest/admin-api.md

```diff
@@ -570,6 +571,8 @@ You can set the `scheme` to `tls`, which means "TLS over TCP".
 
 To use mTLS to communicate with Upstream, you can use the `tls.client_cert/key` in the same format as SSL's `cert` and `key` fields.
 
+Or you can reference SSL object by `tls.client_cert_id` to set SSL cert and key. The SSL object can be referenced only if the `type` field is `client`. Only `cert` and `key` will be used in the SSL object.
```

Comment:
**Otherwise the request will be rejected by APISIX**.

I think we should tell users the result.

Reply:
@soulbird What's the result?